### DISCUSSION TITLE
About block in moose

### DISCUSSION THREAD

**Discussion by Gallasd** [idaholab/moose#21498] [Q&A General] [answered]
I study radiative heat transfer and want to use rays to calculate viewfactors.
Because the current moose does not support meshless yet, I need to create a fluid domain to assist in the  viewfactors calculation.However, the fluid domain is a vacuum, and there is no convection and heat conduction of the fluid.
I checked several examples that come with moose, for example,'moose\modules\heat_conduction\test\tests\radiation_transfer_action\radiative_transfer_action_external_boundary_ray_tracing.i'
[Variables]
  [temperature]
    block = 0
  []
[]

[Kernels]
  [heat_conduction]
    type = HeatConduction
    variable = temperature
    block = 0
    diffusion_coefficient = 5
  []
[]
Here I think it is only heat conduction at block_0, but I found that there is temperature in block_1 in paraview,as shown in the figure
![image](https://user-images.githubusercontent.com/107783970/177289660-4f44584a-91f1-4990-818e-c4d01e5d3e9b.png)
So I want to confirm with you, whether heat conduction is performed in block_1, and whether the temperature of the elements on both sides of block_1 is the reason for interpolation

**Comment by GiudGiud** [comment, marked as answer]
Hello

Heat conduction is not performed in block 1, as you block-restricted it to 0
The only thing you are seeing seems to be the nodal values of temperatures on the boundary between block 0 and block 1.

Guillaume

**Reply by Gallasd** [reply]
ok i see what you mean thanks a lot
